### ISSUE TITLE
A Guide for Acronym

### ISSUE THREAD

**Issue by IIITM-Jay** [alan-turing-institute/the-turing-way#1085]
## Why do we need?
What I felt during my reading is that , "The Turing Way" needs an acronym guide to be mentioned in the book so as to sort the following issues which I encountered: 

*Full forms of some acronyms are written only once when they are used for the first time and in the rest of the book it's missing (of course which is not a healthier task for a writer to mention the full forms every time whenever required and also not providing a better reading view for the readers )  and some don’t have their full forms mentioned too (**like OA and non-OA in one of the pictorial representation** ) which leads to serious consequences of misleading the reader to some other full forms, which can be resolved by writing the full forms of all the acronyms at one place like Appendix as of case in  many of the general books which can be taken as a reference as and when required by any reader. 

-For example, The acronym **RDA: Research Data Alliance** can be misunderstood with ~~Recommended Dietary Allowance (RDA)~~ (famous at worldwide and which is not relevant to the book).
Hence, this misconception needs to be eradicated.

*It’s a hectic and overburden task to mention the full form again and again for the maintainers whenever that acronym is used.

*Always needs to find the full form in the book by the reader wherever it is written , which is not user-friendly and regarded as encumber work to search it again in the book.
 
## What needs to be done?
The best way to get rid of all these issues and problems, is to have a list of all the acronyms used in the book along with their full forms consolidated at one place so as ** to minimize the overload for maintainers of mentioning the full forms of every acronyms , and most hectic to mention the same acronym in the footnotes of every page whenever encountered ,as well as reforming in a more user-friendly manner for readers as well **.
I have listed all the acronyms along with their full forms in my notepad.
Should I proceed further for submitting a PR for review regarding this? Or I have a more better idea that follows. Please have a look.

## A more fabulous idea
The more better idea is to maintain a _“The Turing Way: Dictionary”_ which will include all the acronyms, glossaries, definitions and technical terms, as glossaries too needs to be get updated.

**Comment by malvikasharan** [comment]
Hi @IIITM-Jay, Thanks for opening an issue.

A standard way of using acronym is" "Always write out the first in-text reference to an acronym, followed by the acronym itself written in capital letters and enclosed by parentheses." ([see reference](https://www.e-education.psu.edu/styleforstudents/c2_p9.html)). This is what we use and therefore I don't agree that we need to not use acronyms at all.

Regarding a central location for the acronyms, I think that's a good idea and can be potentially added in the glossary section: https://the-turing-way.netlify.app/glossary/glossary.html. Would like to have a go with that?

**Comment by IIITM-Jay** [comment]
Thanks, @malvikasharan , yes sure I will do that. I will add up as a subsection in the glossary chapter.
Should I make PR regarding the same because I  already have all the acronyms listed with me, which are mentioned so far in the book.

**Comment by malvikasharan** [comment]
Can you please propose this in a new issue? I will be interested to see what others think about the central location for the Acronyms, Gloassary is the right place for it and do they see it fit in the beginning or end of a chapters in the book. @KirstieJane what do you think about it?

**Comment by iiitmjay** [comment]
yes, you were right @malvikasharan . Some of the books have mentioned the list of acronyms at the beginning and some at the last. But as a matter of a guide book, as the case with _"The Turing Way"_ .I think it has to be at the beginning because the book has some sort of a guiding nature . The books which are like a textual material of a high thickness of around 1500 pages , that books are maintaining the same at the last as an appendix.
But still this needs a discussion on the best fit position for the same.
Meanwhile I am opening the same issue newly tagged as " better position for Acronyms" .

**Comment by iiitmjay** [comment]
Opened as suggested by @malvikasharan  in the issue #1086 . Along with acronyms I think we should mention abbreviation , although only one abbreviation came into account used so far i.e., FAIR (**F**undable **A**ccessible **I**nteroperable **R**eusable)  which too conflicts with the 

*Acronym*    **FAIR**: ~~Federation for American Immigration Reform~~ ( not relevant to the book).   

and hence this issue need to be get resolved as early as possible.

**Comment by malvikasharan** [comment]
Thanks Jay. As mentioned in my earlier comment, we are using the standard acronym practice (mention it the first time you use in a chapter and then use the capitalized acronym in the rest), so readers will not consider that as an unrelated acronym. So FAIR as FAIR in this book stands for itself.

**Comment by IIITM-Jay** [comment]
@malvikasharan , Thanks for valuable comment. What about the position for Acronyms (beginning or the end)?  we should wait for others to comment.

**Comment by malvikasharan** [comment]
That discussion can take place in the new issue if you can list that clearly in your issue details. Please use the issue template so that it is easy for us to follow the conversation.
